CHANGELOG.md

```diff
@@ -2352,6 +2431,37 @@ Here is an example of the renaming for the `<ChartsTooltip />` component.
 - [core] Update release instructions as per v7 configuration (#10962) @MBilalShafi
 - [license] Correctly throw errors (#10924) @oliviertassinari
 
+## 6.19.6
+
+_Feb 29, 2024_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+- 🌍 Improve Korean (ko-KR) locale on the Data Grid (#11972) @Luzi
```

Comment:
Nitpick: We need to update these v6 changes as well after changes in the respective PR. 👌